src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -92,6 +98,15 @@ public Person toModelType() throws IllegalValueException {
         }
         final Name modelName = new Name(name);
 
+        if (block == null) {
```

Comment:
Good flow of throwing exceptions! :ok_hand: Maybe you can put this part into a separate method for checking the validity block of the person?

Reply:
I think this is a viable option. I'll consider doing this in a subsequent PR as I believe we should focus on fleshing out the features first before we delve into the coding standards! I'll keep a note on this 👍